Server/serverSlate_test.go

```diff
@@ -1,7 +1,29 @@
 package main
 
-import "testing"
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
 
-func testServerSlate(t *testing.T) {
-	// TODO: Add serverSlate tests
+func TestCalculateLag(t *testing.T) {
+	file, err := ioutil.ReadFile("ocr-data.json")
+	fmt.Println(err)
+	var ocrData HostData
+	json.Unmarshal([]byte(file), &ocrData)
+
+	testStartTimestamp := int64(1597670880210)
```

Comment:
If you are reading some data from a file, you probably should read all of it.

Make a higher level structure, eg:

type LagTestData struct {
  startTimestamp int64
  keyTimestamps []int64
  ocrData HostData
  cameraStartDelay int64
  expectedLag []int64
}

Reply:
added to the same json file.